src/main/scala/cromwell/binding/types/WdlArrayType.scala

```diff
@@ -0,0 +1,20 @@
+package cromwell.binding.types
+
+import cromwell.binding.values.{WdlInteger, WdlArray}
+import spray.json.JsArray
+
```

Comment:
Let me double check but I think since `*` and `+` utilize the concept of Arrays a little bit I think that's why it's in here, but that may not actually be true.